x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -110,6 +118,11 @@ public void testGroupByOrderByNonGrouped() {
                 verify("SELECT MAX(int) FROM test GROUP BY text ORDER BY bool"));
     }
 
+    public void testGroupByOrderByAliasedInSelectAllowed() {
+        LogicalPlan lp = accepted("SELECT text t FROM test GROUP BY text ORDER BY t");
```

Comment:
You don't group by the alias `t` but you use `text`, was that the intention? Because here you use the alias: https://github.com/elastic/elasticsearch/pull/34773/files#diff-3d7c2dc164e7f948316c325f88c12a3fR39

Reply:
The verifier query is correct, the sql isn't (it's a typo). Fixing this in a follow-up commit.